### DISCUSSION TITLE
Questions about Relative Permeability Estimation

### DISCUSSION THREAD

**Discussion by MingyueTianya1** [PMEAL/OpenPNM#2482] [Q&A]
Hello Professor ，
         when I use SNOW2 to extract the network, I got the GeneIcnetwork and the extracted network tag is pore.xmin，pore.xmax，pore.ymin，etc. However, when I run the `rp = op.algorithms.metrics.RelativePermeability(network=pn)`， I will report the error of `KeyError: 'pore.front'` or`KeyError: 'pore.top'`.I checked the Relative Permeability Estimation.py.The label that it needs to run is pore.left, pore.right, pore.top, what should I do?

**Comment by jgostick** [comment]
Which version are you using?  Can you report the printout of ``op.__version__``?

**Reply by MingyueTianya1** [reply]
I used openpnm 2.8.2
